src/code/z_lib.c

```diff
@@ -300,9 +300,14 @@ f32 Math_Vec3f_DiffY(Vec3f* a, Vec3f* b) {
     return b->y - a->y;
 }
 
-s16 Math_Vec3f_Yaw(Vec3f* a, Vec3f* b) {
-    f32 dx = b->x - a->x;
-    f32 dz = b->z - a->z;
+/**
+ * @param origin Position of the origin, the location from which to look at the target `point`
+ * @param point Position of the target point, in the same space as `origin`
+ * @return The yaw towards `point` when at `origin`, assuming +z is forwards.
+ */
+s16 Math_Vec3f_Yaw(Vec3f* origin, Vec3f* point) {
```

Comment:
Could you also add the yaw is the clockwise, counterclockwise, or closest angle? (I'm not sure which it is)

Reply:
Well, it wraps around the +y axis like usual (according to the right hand rule)

I feel like making this explicit would belong in a larger note / trigonometry crash course 🤔

Though you could argue angles aren't always directed as expected, like pitch in camera being sometimes wrapped around +x and sometimes around -x ...

Reply:
Sorry, this is your second comment I reply with basically "details exceed the scope of just this function"

I can try to write a `docs/rotations.md` document or similar but idk about what we want to put in docs/

Reply:
Yeah, no worries. I'm just brainstorming how to make these angles/yaws clearer. If you're playing Link in game, there's not really any clear way to tell which directions are the "cardinal axis" i.e. which direction is +z and +x? (Except maybe the minimap). +y is easy because that's just "up". so by the right hand rule, it would be counterclockwise looking top-down?

Pitch is actually clear to me because up/down is well defined.

so `+z` would be 0 degrees, but is +x facing left or right? I would say left by the Right Hand Rule? I suppose you can infer that. So left/+x would be 90 degrees? And -z would be 180 degrees? And -x would be 270 degrees? Do I have those numbers right?

Reply:
That is correct afaik yeah.